antora.yml

```diff
@@ -6,7 +6,7 @@ nav:
 - modules/ROOT/nav.adoc
 asciidoc:
   attributes:
-    project-github: redpanda-data/connect
+    project-github: redpandadata/connect
```

Comment:
I might be misunderstanding what this is for, but shouldn't this remain as `redpanda-data/connect`?

Reply:
I agree. Let’s keep this as is. We may need it in future.

@asimms41 we can stop using this attribute in Docker URLs and replace it with `redpandadata/connect`